src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -120,6 +133,12 @@ export class Lilies extends CoreGauge {
 
 	}
 
+	private checkAfflatusAvailability() {
+		if (this.lilyGauge.value !== 0 && !this.bloodLilyGauge.capped) {
```

Comment:
Not sure about this function name, and it could be inlined with the hook anyway, but there's a getter to check when the gauge is empty instead of a specific value.

```suggestion
		if (!this.lilyGauge.empty && !this.bloodLilyGauge.capped) {
```